tests/conftest.py

```diff
@@ -165,11 +183,11 @@ def selenium(selenium: WebDriver) -> WebDriver:
     return selenium
 
 
-def create_user(request_session: requests.Session, server: str, auth: Tuple[str, str]):
+def create_user(request_session: requests.Session, server: str, auth: Auth):
     # check if user already exists before creating
-    r = request_session.get(f"{server}/", auth=auth)
+    r = request_session.get(server, auth=auth)
     if "user" not in r.json():
-        create_account_url = f"{server}/accounts/{auth[0]}"
+        create_account_url = urljoin(f"{server}/", f"accounts/{auth[0]}")
```

Comment:
I'm having doubts about this one. Everywhere in the kinto repos we do the opposite (no trailing slash on server url and leading slash on paths)

I would recommend normalize `server` in the fixture code, to make sure it has no trailing slash, and then concatenating strings without `urljoin()`. It's not really like filepaths where delimiters can differ depending on the OS, if we don't need the smart features of `urljoin()` then I think concatenating strings is more straightforward to read

Reply:
https://github.com/mozilla-services/kinto-dist/pull/1970/commits/005eb314aa7a6883f3e39484344bb08ade3c094f